bookkeeper-server/pom.xml

```diff
@@ -178,10 +178,24 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+        <groupId>org.apache.hadoop</groupId>
+        <artifactId>hadoop-minikdc</artifactId>
```

Comment:
Scope is "test" and it is needed in order to have a working Kerberos server. I'm using the one used in ZooKeeper.
Maybe we can switch to http://directory.apache.org/kerby/ if preferred (ZooKeeper uses both of them for different branches)